[W16-1] unsure about product name

jwl1997 opened the pull request:
I am so sorry for the late PR, because we just restructured our team.
@Yinya998
@daDangminh
@yonggqiii
@JillWangJill

ronghualui commented:
Comments for v1.1:
First of all, please change the PR name to the name of the product.

Please update the description of your project:
![image](https://user-images.githubusercontent.com/17693908/54068198-f977b900-4284-11e9-9331-a19e8e85bef1.png)

README page:
Generally no issue, but right now, your travis badge is not pointing to your repository (build:unknown), because of the change in your repository name so this needs to be updated.

About Us:
No issues found.

User Guide/Developer Guide:
We will give you the comments in a separate document to avoid cluttering this PR.

yinya998 committed:
travis

yinya998 committed:
travis

JiaruiWang-Jill committed:
Merge pull request #193 from yinya998/master

fix bugs reported in dry run:

yonggqiii committed:
Users can now block certain timings to meet.

yonggqiii committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W16-1/main

yonggqiii committed:
Checkstyle

yonggqiii committed:
Added space after coma lmao

yinya998 committed:
update developer guide

yinya998 committed:
typo

yinya998 committed:
typo

jwl1997 committed:
listE test cases

yinya998 committed:
travis

JiaruiWang-Jill committed:
Merge pull request #195 from yinya998/master

update developer guide

yinya998 committed:
fix photo command in jar

these three lines in fileutil cannot be deleted ; otherwise the jar cannot add photo

yinya998 committed:
format

yinya998 committed:
typo

yinya998 committed:
Merge pull request #196 from yinya998/master

fix photo command bug in jar

JiaruiWang-Jill committed:
deleteR, listR command. Reminder update

JiaruiWang-Jill committed:
Merge branch 'master' into reminderCommand

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
Merge branch 'reminderCommand' of https://github.com/JillWangJill/main into reminderCommand

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
gg

jwl1997 committed:
test cases

jwl1997 committed:
bug fix

jwl1997 committed:
bug fix

jwl1997 committed:
bug fix

jwl1997 committed:
Merge pull request #194 from jwl1997/master

integration test

JiaruiWang-Jill committed:
travis test and merge

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
Update testingName.json

JiaruiWang-Jill committed:
Merge pull request #197 from JillWangJill/reminderCommand

deleteR, listR command. Reminder update

yonggqiii committed:
Merge pull request #198 from yonggqiii/master

Users can now use block/ to signify meeting only in certain hours.

JiaruiWang-Jill committed:
listFR command, deal with swtich

JiaruiWang-Jill committed:
Merge branch 'reminderCommand' of https://github.com/JillWangJill/main into reminderCommand

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
Merge pull request #199 from JillWangJill/reminderCommand

listFR and switch command

jwl1997 committed:
add test cases

jwl1997 committed:
bug fix

jwl1997 committed:
Merge pull request #200 from jwl1997/master

add test cases

jwl1997 committed:
Merge pull request #18 from CS2103-AY1819S2-W16-1/master

update my own repo

jwl1997 committed:
update developerguide

jwl1997 committed:
picture

jwl1997 committed:
add test cases

jwl1997 committed:
bug fix

jwl1997 committed:
bug fix

jwl1997 committed:
bug fix

jwl1997 committed:
Merge pull request #201 from jwl1997/master

update developer guide

jwl1997 committed:
Update DeveloperGuide.adoc

jwl1997 committed:
Update DeveloperGuide.adoc

yonggqiii committed:
MeetCommand basic functionalities test

yonggqiii committed:
checkstyle

yonggqiii committed:
checkstyle

JiaruiWang-Jill committed:
Command improvement and user guide update

JiaruiWang-Jill committed:
userguide

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
userguide

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
checkstyle

JiaruiWang-Jill committed:
Merge pull request #203 from JillWangJill/reminderCommand

User guide + reminder improve

JiaruiWang-Jill committed:
userguide

JiaruiWang-Jill committed:
user

JiaruiWang-Jill committed:
userguide update

JiaruiWang-Jill committed:
Merge pull request #204 from JillWangJill/reminderCommand

userguide update

yonggqiii committed:
Merge pull request #205 from yonggqiii/master

MeetCommand basic functionalities test

yonggqiii committed:
More MeetCommand tests

yonggqiii committed:
MeetCommand achieves 100% coverage

yonggqiii committed:
remove unused import

yonggqiii committed:
Checkstye for MeetCommandBuilder

yonggqiii committed:
Merge pull request #206 from yonggqiii/master

More MeetCommand tests

jwl1997 committed:
Merge pull request #19 from CS2103-AY1819S2-W16-1/master

update my own repo

jwl1997 committed:
some modification

jwl1997 committed:
Merge pull request #207 from jwl1997/master

more change in connect command

jwl1997 committed:
Merge pull request #20 from CS2103-AY1819S2-W16-1/master

Merge pull request #207 from jwl1997/master

jwl1997 committed:
Update DeveloperGuide.adoc

yonggqiii committed:
Test coverage for MeetCommandParser hit 100%

yonggqiii committed:
More MeetCommandParser tests

yonggqiii committed:
I really want to know what the javadoc comments could possibly be for.

yonggqiii committed:
How could they possibly ask me for a javadoc comment for a private method, i do not know.

yonggqiii committed:
Merge pull request #208 from yonggqiii/master

Test coverage for MeetCommandParser hit 100%

yonggqiii committed:
Tested MeetCommandParser

yonggqiii committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W16-1/main

yonggqiii committed:
Added basic meet command user flow

yonggqiii committed:
Changed line endings

yonggqiii committed:
Merge pull request #209 from yonggqiii/master

More meet command parser tests, and additions to UserGuide

yonggqiii committed:
Completed meet command user guide

yonggqiii committed:
Testing out link to UG

yonggqiii committed:
Dos2unix

yonggqiii committed:
Remove trailing whitespace

yinya998 committed:
change sample data to demo

yinya998 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W16-1/main

yinya998 committed:
update user guide and developer guide for find findE photo

yinya998 committed:
Merge pull request #211 from yinya998/master

update user guide, developer guide

daDangminh committed:
Merge pull request #11 from CS2103-AY1819S2-W16-1/master

update

daDangminh committed:
update

daDangminh committed:
bug fix

daDangminh committed:
Merge pull request #212 from daDangminh/export

bug fix

yinya998 committed:
fix grammar error in user guide and developer guide

yonggqiii committed:
Added some stuff for meeting planner implementation

yinya998 committed:
format uesr guide

yinya998 committed:
format adoc userguide

yinya998 committed:
format *@gmail

yinya998 committed:
Merge pull request #210 from yonggqiii/master

Completed UG for meet command

yinya998 committed:
format

yinya998 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W16-1/main

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

yonggqiii committed:
Removed trailing commas

yonggqiii committed:
Removed displayP from UserGuide

yonggqiii committed:
Merge pull request #213 from yonggqiii/master

Removed trailing commas

yinya998 committed:
add photo command test

yinya998 committed:
format

yinya998 committed:
Merge pull request #214 from yinya998/master

add photo test and photo command test

JiaruiWang-Jill committed:
update developer guide and ppp

JiaruiWang-Jill committed:
Merge pull request #215 from JillWangJill/reminderCommand

update developer guide and ppp

JiaruiWang-Jill committed:
fix style bugs

yinya998 committed:
Merge pull request #216 from JillWangJill/reminderCommand

fix style bugs

yinya998 committed:
add sample event data

yinya998 committed:
ppp

yinya998 committed:
format

yinya998 committed:
format

yinya998 committed:
Merge branch 'master' into master

yinya998 committed:
format

yinya998 committed:
Merge pull request #217 from yinya998/master

format

yinya998 committed:
format

yinya998 committed:
format

yinya998 committed:
add sample event data

yinya998 committed:
Merge branch 'master' of https://github.com/yinya998/main

yinya998 committed:
format

yinya998 committed:
format

yinya998 committed:
Merge branch 'master' into master

yonggqiii committed:
Sequence diagram for MeetCommand

yonggqiii committed:
included file extension for image lmao

yonggqiii committed:
Edited sequence diagram for meet command

yonggqiii committed:
Merge branch 'master' into master

yonggqiii committed:
Merge pull request #219 from yonggqiii/master

MeetCommand DG updates

yonggqiii committed:
Fixed meetcommand bug and events equality check for connected persons

yonggqiii committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W16-1/main

yonggqiii committed:
UG and DG reflects changes to meet command

yonggqiii committed:
Checkstyle

yonggqiii committed:
A bunch of changes to devguide and meetcommand implementation

yonggqiii committed:
Possibly the most trivial checkstyle ever, and also shortened the height of MeetCommandLogic 1 as it was pointlessly taking too much space on top and below.

yonggqiii committed:
Another pointless checkstyle, but also cool new images for DevGuide

yonggqiii committed:
More images for developer guide

yonggqiii committed:
Added a code snippet in the DeveloperGuide for clearer understanding of the reduce method used for stream reduction

yonggqiii committed:
Cleaned up MeetCommand and wrote more DevGuide for MeetCommand implementation

yonggqiii committed:
Forgot to label y as green

yonggqiii committed:
Another diagram for MeetCommand implementation in DevGuide

yonggqiii committed:
For some reason the AboutUs page points to JohnDoe for yonggqiii

yonggqiii committed:
Forgot to save MeetCommandLogic6.png

yonggqiii committed:
Even more diagrams. Am I going to keep to the 10 page limit?

yonggqiii committed:
Merge pull request #220 from yonggqiii/master

Fixed some meet command bugs and forced equality check to compare persons in events. Also, additions to DevGuide and UserGuide too.

yonggqiii committed:
Edited all images and included them in DevGuide

yonggqiii committed:
Merge pull request #218 from yinya998/master

format yinya ppp

yonggqiii committed:
Completed logic behind reduce function in MeetCommand implementaiton in DevGuide

yonggqiii committed:
New UI

yonggqiii committed:
Merge pull request #221 from yonggqiii/master

Completed logic behind MeetCommand's reduce usage in DG.

yinya998 committed:
format

yinya998 committed:
format

JiaruiWang-Jill committed:
data added

JiaruiWang-Jill committed:
update files and test

JiaruiWang-Jill committed:
sample data modified

JiaruiWang-Jill committed:
update

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
space delete

yonggqiii committed:
Reduced UserGuide length for Meet Command

yonggqiii committed:
Further reductions to meeting planner portion

yonggqiii committed:
Merge pull request #225 from yonggqiii/master

Reduced UserGuide length for Meet Command

yonggqiii committed:
Fixed issue #224

yonggqiii committed:
Checkstyle

yonggqiii committed:
Merge pull request #226 from yonggqiii/master

Fixed issue #224

yonggqiii committed:
Edited my ppp

yonggqiii committed:
Merge pull request #227 from yonggqiii/master

Edited my ppp

JiaruiWang-Jill committed:
import sequence

yonggqiii committed:
Devguide for testing meet command

yonggqiii committed:
Edited dev guide

JiaruiWang-Jill committed:
developer guide update

yonggqiii committed:
Merge pull request #228 from yonggqiii/master

meow

jwl1997 committed:
Merge pull request #21 from CS2103-AY1819S2-W16-1/master

update my repo

yonggqiii committed:
Connect and Disconnect now support undo/redo

JiaruiWang-Jill committed:
Update testingName.json

jwl1997 committed:
bug fix

yonggqiii committed:
Merge pull request #229 from yonggqiii/master

Connect and Disconnect now support undo/redo

JiaruiWang-Jill committed:
update DG

JiaruiWang-Jill committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W16-1/main into reminderCommand

JiaruiWang-Jill committed:
Merge branch 'reminderCommand' of https://github.com/JillWangJill/main into reminderCommand

JiaruiWang-Jill committed:
DG update

JiaruiWang-Jill committed:
DG

JiaruiWang-Jill committed:
Merge pull request #222 from JillWangJill/reminderCommand

update files and test

yinya998 committed:
add sample data

yinya998 committed:
Merge branch 'master' of https://github.com/yinya998/main

yinya998 committed:
Merge branch 'master' into master

yinya998 committed:
format

yinya998 committed:
Merge pull request #231 from yinya998/master

add sample data

yonggqiii committed:
Final edits to Yong Qi's documentation

yinya998 committed:
update sample event

yonggqiii committed:
Merge pull request #232 from yonggqiii/master

Final edits to Yong Qi's documentation

yinya998 committed:
Merge pull request #233 from yinya998/master

update sample event

yinya998 committed:
update sample data

yonggqiii committed:
Block achieved 100% coverage

yonggqiii committed:
Changed import order

yonggqiii committed:
Merge pull request #234 from yonggqiii/master

Block coverage at 100%

JiaruiWang-Jill committed:
update sequence diagram and reminder bug fix

JiaruiWang-Jill committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W16-1/main into reminderCommand

jwl1997 committed:
update portfolio

jwl1997 committed:
Update jwl1997.adoc

yonggqiii committed:
Switch Command at 100% coverage

yonggqiii committed:
Changed import order

yonggqiii committed:
Checkstyle

daDangminh committed:
ppp

JiaruiWang-Jill committed:
update manual test

daDangminh committed:
Merge pull request #236 from daDangminh/export

ppp

daDangminh committed:
ppp

yonggqiii committed:
Merge pull request #235 from yonggqiii/master

SwitchCommand tests

JiaruiWang-Jill committed:
image size

jwl1997 committed:
Merge branch 'master' into master

daDangminh committed:
devguide

JiaruiWang-Jill committed:
update data

daDangminh committed:
Merge pull request #237 from daDangminh/export

ppp

JiaruiWang-Jill committed:
update

JiaruiWang-Jill committed:
space

daDangminh committed:
userguide

JiaruiWang-Jill committed:
travis

daDangminh committed:
Merge pull request #239 from daDangminh/export

userguide

JiaruiWang-Jill committed:
Merge pull request #238 from JillWangJill/reminderCommand

sample data modify

yinya998 committed:
update

jwl1997 committed:
Update UserGuide.adoc

daDangminh committed:
Merge pull request #12 from CS2103-AY1819S2-W16-1/master

update

jwl1997 committed:
Update UserGuide.adoc

jwl1997 committed:
Update UserGuide.adoc

daDangminh committed:
bug fix

jwl1997 committed:
Merge pull request #241 from daDangminh/export

bug fix

JiaruiWang-Jill committed:
duplicate reminder bug

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
travis

jwl1997 committed:
Revert "bug fix"

jwl1997 committed:
Merge pull request #243 from CS2103-AY1819S2-W16-1/revert-241-export

Revert "bug fix"

jwl1997 committed:
Revert "Revert "bug fix""

yinya998 committed:
update a name

JiaruiWang-Jill committed:
travis

JiaruiWang-Jill committed:
test fix

JiaruiWang-Jill committed:
traivis

jwl1997 committed:
Merge pull request #244 from CS2103-AY1819S2-W16-1/revert-243-revert-241-export

Revert "Revert "bug fix""

yinya998 committed:
Merge pull request #245 from yinya998/patch-1

update a name

jwl1997 committed:
Merge pull request #242 from JillWangJill/reminderCommand

duplicate reminder bug

jwl1997 committed:
Update DeveloperGuide.adoc

jwl1997 committed:
Update DeveloperGuide.adoc
